src/test/java/core/basesyntax/service/ContentGeneratorImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.impl.ContentGeneratorImpl;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ContentGeneratorImplTest {
+    private static ContentGenerator contentGenerator;
+    private static Map<Fruit, Integer> fruitIntegerMap;
+    private static String report;
+
+    @BeforeClass
+    public static void setUp() {
+        fruitIntegerMap = new HashMap<>();
+        fruitIntegerMap.put(new Fruit("banana"), 152);
+        fruitIntegerMap.put(new Fruit("apple"), 90);

```

Comment:
this relates to `generateContent_validData_ok` only
so let's move this logic to the test and make `fruitIntegerMap` local variable